LIFXMasterApp.groovy

```diff
@@ -720,7 +720,7 @@ Map<String, List> deviceSetColorTemperature(com.hubitat.app.DeviceWrapper device
 
 Map<String, List> deviceSetIRLevel(com.hubitat.app.DeviceWrapper device, Number level, Boolean displayed, duration = 0) {
     def actions = makeActions()
-    actions.commands << makeCommand('LIGHT.SET_INFRARED', [irLevel: value == scaleUp100(level)])
+    actions.commands << makeCommand('LIGHT.SET_INFRARED', [irLevel: value = scaleUp100(level)])
```

Comment:
No idea what I was thinking there! Well spotted.

Since my LIFX+ bulb is currently packed away, can you try it without the assignment to value (which isn't used at all) 

ie 

`actions.commands << makeCommand('LIGHT.SET_INFRARED', [irLevel: scaleUp100(level)])`

I would expect that to work just as well.

Reply:
I'll give that a go tonight (when I can tell whether the IR level has been successfully changed), and report back

Big thanks for putting this all together! I had started reading up on it, to throw something together in my spare time - but never got very far into it

Reply:
It's been a labour of love - and maybe a bit of hate at times :)

Reply:
LOL -- and yes, the further simplification works as expected - amended this PR